plugins/woocommerce/tests/php/includes/wc-product-functions-test.php

```diff
@@ -128,4 +128,72 @@ public function get_customer_id() {
 			remove_filter( 'woocommerce_adjust_non_base_location_prices', '__return_false' );
 		}
 	}
+
+	/**
+	 * @testdox Test 'wc_get_price_including_tax'.
+	 *
+	 * @testWith [true, true]
+	 *           [true, false]
+	 *           [false, true]
+	 *           [false, false]
+	 *
+	 * @param bool $prices_include_tax Whether entered prices are inclusive of tax.
+	 * @param bool $is_vat_exempt      Whether the VAT is exempted for customer.
+	 */
+	public function test_wc_get_price_including_tax( $prices_include_tax, $is_vat_exempt ) {
+		// Set VAT exempt and Mock prices_include_tax.
+		WC()->customer->set_is_vat_exempt( $is_vat_exempt );
```

Comment:
Does this setting also need a cleanup or is the order of execution of `testWith` annotation guaranteed, so it will end up with neutral (false) value?

Reply:
Thanks for noticing this @peterfabian. Yes, cleanup is needed I have added the cleanup code. Could you please check it once? Thanks.